addons/bsp_importer/bsp_reader.gd

```diff
@@ -252,6 +251,13 @@ var plane_normals : PackedVector3Array
 var plane_distances : PackedFloat32Array
 var model_scenes : Dictionary = {}
 
+var _unit_scale: float = 1.0
+
+var inverse_scale_fac: float = 16.0:
```

Comment:
Probably doesn't matter, but perhaps this should be the default 32.

Reply:
oh good catch! i just copied it over from my code and missed this one